MangaManager/src/DynamicLibController/models/IMetadataSource.py

```diff
@@ -35,10 +37,42 @@ def update_people_from_mapping(people: list[object], mapping, comicinfo: ComicIn
             for map_role in mapping:
                 if map_role == role:
                     for fields in mapping[map_role]:
-                        comicinfo.set_by_tag_name(fields, name.strip())
+                        old_name = comicinfo.get_by_tag_name(fields.strip())
+                        if old_name and old_name.strip() != "":
+                            name = old_name.strip() + ", " + name.strip()
+                        comicinfo.set_by_tag_name(fields.strip(), name.strip())
 
             print(f"No mapping found for: {name} as {role}")
 
+    def strip_description_html_tags(summary):
```

Comment:
You're going to need to document how this works and how it can be customized or adapted.